CHANGELOG.md

```diff
@@ -2,9 +2,15 @@
 
 ## Unreleased
 
+## v0.91.0
```

Comment:
```suggestion
## v0.91.1
```

Reply:
I pushed this changed to gh, but it is not persisting to this PR.